src/components/layout/useGradient.tsx

```diff
@@ -0,0 +1,12 @@
+import { MantineGradient } from '@mantine/core';
+import { useColorTheme } from '../../tools/color';
+
+export const usePrimaryGradient = (): MantineGradient => {
```

Comment:
This function should be used more often. We make the same thing in many other places